homeassistant/components/androidtv/media_player.py

```diff
@@ -84,6 +87,74 @@
     {vol.Required(ATTR_ENTITY_ID): cv.entity_ids, vol.Required(ATTR_COMMAND): cv.string}
 )
 
+# To be used for config validation
+VALID_PROPERTIES = ("audio_state", "media_session_state")
```

Comment:
It would be nice if the library made all the valid properties, strings and conditions available for import so that the user of the library can validate the rules passed as needed easily.